### ISSUE TITLE
How to access store/component data inside Hooks?

### ISSUE THREAD

**Issue by mubaidr** [ankurk91/vue-flatpickr-component#61]
```
[ ] Bug report => search github for a similar issue or PR before submitting
[x] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* flatPickr version : 4.2.4
* Vue.js version : 2.5.13
* Browser name and version : Chrome
* This package version : 6.1.0

**Current behavior**
I want to access component data inside `onDayCreate` hook.

**Expected behavior**
Don't know if any

Is there any word around for this?

**Comment by ankurk91** [comment]
This is same as
https://chmln.github.io/flatpickr/events/#ondaycreate

**Comment by mubaidr** [comment]
I know the hook is named `onDayCreate` that is why I have it written inside issue report. 

**Is it possible to access store/component data inside Hooks?**

**Comment by mubaidr** [comment]
Anyway, solved it by using Async function. 

TLDR; Yes, it is possible.
